### DISCUSSION TITLE
Where is react-static 8? The state of react-static -- and its inevitable death.

### DISCUSSION THREAD

**Discussion by SleeplessByte** [react-static/react-static#1661] [General]
When my company took over development of react-static, we quickly solved a lot of outstanding issues and a highly active team of contributors pumped out even more fixes and upgrades to the current codebase. This was great. 

_Then COVID happened._

During the first months of COVID-19, we lost 80% of our _recurring_ income, most of whomstve were running on `react-static`. Not only did we have to scramble for more work, the work we found was very different and did not need `react-static` at all. You may have seen me comment on various issues that we were struggling during COVID keeping this alive, because we had an extremely high workload to keep the company alive, without a single paying client funding this work.

_Fast-forward to 2022._

We are doing great as a company (with more work offers than we can take on), but not a single company that left us came back after COVID. The majority of which were US-based and are still struggling, whereas our EU-based clients (now > 75%) are all doing very well. Because there is no funding for react-static, and there is literally not a single reason for us to keep investing in it, I'm going to let go of the project.

I understand this is still a helpful tool for some people, and we have no intentions of purging it from existence, but there is just no economic viability. 

For the projects where we normally had used `react-static`, we now use [remix](https://remix.run/), [astro](https://astro.build/), and [nextjs](https://nextjs.org/) for others. 

Feel free to leave your thoughts here, but if you have time to spare, instead, see what you can do to help Ukraine, including supporting those Russians who are against the war.

**Comment by mcnaveen** [comment]
Thank you for coming upfront with this level of honesty. Good luck with your company :heart:

**Comment by HarveyPeachey** [comment]
Is there a reason why issues have been disabled? I'm migrating away from react-static but when I first started there was a lot of useful information about the SSG in either open and closed out issues, is there no maintenance like read-only view for issues? I also recall there being conversations about migrating to other SSG frameworks which would prove useful

**Reply by SleeplessByte** [reply]
Ah. That's not great indeed. I'll re-enable it for you and add a issue template to convey this information. 

Apologies!

**Reply by SleeplessByte** [reply]
Done :)

**Reply by HarveyPeachey** [reply]
No worries, it was this comment I was referring to https://github.com/react-static/react-static/issues/1203#issuecomment-626361725, however, steps might be different now due to Gatsby's updates since the comment but should be a good starting point. Thank you!

**Comment by marekpiechut** [comment]
Thanks for sharing the news. It's really sad to see it go 😢. I really like react-static. I find it so much easier to use and reason about when compared to stuff like Gatsby. It was just this nice balance between power and simplicity.

Thanks for all the time you have put into the project. Wish you all the best 👋!

**Reply by SleeplessByte** [reply]
I also really like `react-static`, despite the absolute pain that is `react-universal-component`. I personally do think that there _is_ a place in the space once react's server components is in production, because it will take away the part that is the most painful (universal/hydration). 

By that point I do expect people to be able to react-staticify something simple such as a create-react-app template. 

We'll see what the future holds!

**Comment by ghost** [comment]
i will never forget this project

**Comment by ghost** [comment]
lets combat framework death by using react static even though its deprecated

**Reply by daviskeene** [reply]
I tried but I cannot build my site 😢

**Comment by ghost** [comment]
Sent from Proton Mail mobile

-------- Original Message --------

**Comment by ishan28mkip** [comment]
Why not transfer the ownership to contributors instead of shutting it down?

**Reply by SleeplessByte** [reply]
Because there are no active contributors

**Comment by dkleriga** [comment]
Hi, I'd like to contribute to this project and possibly take it over. NextJS doesn't work for me and is far more trouble than it's worth. Is this an option? Happy to fork it if you prefer, but I'd like to revive the main project as I found it to be the best version of static react

**Reply by SleeplessByte** [reply]
You can fork it. I will protect the domain and repo name and npm package name for now.
